python/libraries/SystemIO.py

```diff
@@ -8,6 +10,15 @@ async def inp(question):
     with ThreadPoolExecutor(1, "AsyncInput") as executor:
         return await asyncio.get_event_loop().run_in_executor(executor, input, question)
 
+def run(command):
+    can_run = os.environ.get("COMMAND_ALLOW") == "true"
+    if not can_run:
+        return 1
+
+    return os.system(COMMAND_ALLOW)
 
 def _values():
-    return {"inp": ("str", n_cmd_type.with_typevars(["str"]))}
+    return {
+        "inp": ("str", n_cmd_type.with_typevars(["str"])),
+        "run": ("str", "int"),
```

Comment:
If this just returns 0/1, perhaps we can make this a bool?

Also, it should be a `cmd` since it has side effects

Reply:
Perhaps

```suggestion
        "run": ("str", n_cmd_type.with_typevars(["bool"])),
```

Reply:
Fair enough